apisix/discovery/nacos/init.lua

```diff
@@ -225,13 +224,20 @@ local function iter_and_add_service(services, values)
         else
             up = conf
         end
+        local nacos_name_form_args = (up.discovery_args and up.discovery_args.name)
+             or default_nacos_name
+        local nacos_name = nacos.name or default_nacos_name
+        if   nacos_name ~= nacos_name_form_args then
```

Comment:
a lot of code style problems are there, please pay attention and fix elsewhere
```suggestion
        if nacos_name ~= nacos_name_form_args then
```